fitbenchmarking/fitbenchmarking
# 763 change bounds test

#### Description of Work

Fixes #763

This change adds a note to the documentation that bumps may not respect bounds for sasview problems - I have contacted members of the SANS group for more insight on why this might be the case so more detail can be added to the docs/the problem can be fixed. This will be addressed in a separate PR.

#### Testing Instructions

1. Do all controllers pass tests as expected?

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
